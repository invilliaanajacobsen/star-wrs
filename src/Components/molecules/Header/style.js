import styled from 'styled-components';
import Row from 'react-bootstrap/Row'

import { device } from '../../../styles.js';


export const Line = styled(Row)`
	display: flex;
	align-items: center;
	justify-content: center;
	height: 50%;
		
	@media ${device.mobile} { 
		height: 40%;
	}
	@media ${device.mobilexs} { 
		height: 31s%;
	}

`;

export const Heading = styled.div`
	width: 40%;
	height: 100%;	
	@media ${device.mobile} { 
		width:40%;
		height: 50%;
	}
	@media ${device.laptop} { 
		width:25%;
	}
`;

