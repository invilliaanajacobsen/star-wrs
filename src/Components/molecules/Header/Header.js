import React from 'react';
import { Line, Heading } from './style.js';

import Icon from "../../atoms/Icon"

function Header() {
	return (
		<Line>
			<Heading>
				<Icon icon="starwars" height="auto" width="auto" fill="#ffffff" alt="starwars icon" />
			</Heading>
		</Line>
	);
}

export default Header;
