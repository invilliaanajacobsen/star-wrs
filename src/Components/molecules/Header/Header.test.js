import React from "react";
import { shallow, } from "enzyme";

import Header from "./Header";

describe("Header component", () => {
	it("should render Header match snapshot", () => {
		expect(shallow(<Header />)).toMatchSnapshot();
	});

});
