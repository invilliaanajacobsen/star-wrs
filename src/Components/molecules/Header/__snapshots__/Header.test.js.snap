// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header match snapshot 1`] = `
<Styled(Component)>
  <styled.div>
    <Icon
      alt="starwars icon"
      fill="#ffffff"
      height="auto"
      icon="starwars"
      width="auto"
    />
  </styled.div>
</Styled(Component)>
`;
